scan

```diff
@@ -80,13 +80,15 @@ def scan_domains(scanners, domains):
             'writer': scanner_writer
         }
 
-    # For each domain, run through each scanner.
```

Comment:
A note that I just tackled parallelization in #22, which did involve flipping the loop after all. However, I'm not sure of the effect this parallelization will have on phantomjs. Happy to work with you to figure it out.